.github/workflows/test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.15 # https://github.com/sindresorhus/refined-github/issues/2900
+          node-version: 14
```

Comment:
AVA refuses to run `.mjs` tests on Node 12. Related: https://github.com/avajs/ava/issues/2432